### PR TITLE
Markup changes to wifi and stylesheet improvements

### PR DESCRIPTION
**Pull request by jbeta51**

### CODE REVIEW

File: pkg/sos/html/wifi.html [hunk 1]
```diff
@@ -0,0 +1,154 @@
+<head>
+	<script>
```
Comment (GanShun) [inline pkg/sos/html/wifi.html]: We should start thinking about separating out the javascript into different files. I'm not sure we want to keep embedding it in the html. This is fine for now though. Not a blocker.

File: cmds/wifi/server.go [hunk 2]
```diff
@@ -254,6 +255,7 @@ func (ws WifiServer) buildRouter() *mux.Router {
 	r.HandleFunc("/", ws.displayStateHandle).Methods("GET")
 	r.HandleFunc("/refresh", ws.refreshHandle).Methods("POST")
 	r.HandleFunc("/connect", ws.connectHandle).Methods("POST")
+	r.PathPrefix("/css/").Handler(http.StripPrefix("/css/", http.FileServer(http.Dir("/etc/sos/html/css"))))
```
Comment (GanShun) [inline cmds/wifi/server.go]: What happens if the file doesn't exist?

Comment (jbeta51) [reply cmds/wifi/server.go]: It just won't load the css. The html will persist though and you can still navigate the different services

File: pkg/sos/html/wifi.html (hunk 1 above)
Comment (jbeta51) [reply pkg/sos/html/wifi.html]: I agree. That's the next step after getting upspin up to speed with the other pages

File: cmds/wifi/server.go (hunk 2 above)
Comment (GanShun) [reply cmds/wifi/server.go]: that's good. This is good to go

**Pull request merged**